### DISCUSSION TITLE
Upgrade to Nuxt3？

### DISCUSSION THREAD

**Discussion by kuizuo** [hoppscotch/hoppscotch#2864] [General]
Any idea to consider upgrading Nuxt3?

I know it's going to be a lot of work, I'm just asking about the intent to upgrade。

**Comment by liyasthomas** [comment]
We're currently on Vue v3.x + Vite v3.x stack. Nuxt dependency has been completely removed since https://github.com/hoppscotch/hoppscotch/commit/8b300fab5d8a7fa3dd5a73e5e44d71018d0defec.
